.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Pull Request Guidelines
+
+* * *
+
+Before you submit a pull request, check that it meets these guidelines:
+
+1. Read getting started in [CONTRIBUTING.md](CONTRIBUTING.md#get-started).
+1. If the pull request adds functionality, it is good practice to also add tests for the functionality.
+1. Update the [CHANGELOG](CHANGELOG.md) in accordance with the nature of your Pull Request.
+1. The pull request should work for C++, emscripten, web browser, Python 3.6, 3.7 and 3.8.
+   A continuous integration job will run tests for you.
+   Check green check mark on [https://github.com/NLESC-JCER/cpp2wasm/pulls](https://github.com/NLESC-JCER/cpp2wasm/pulls)
+   and make sure that the tests pass.
+
+If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.
+
+* * *
+
+Closes {Link to the corresponding issue}
```

Comment:
I'm not a fan of automated linking of issues, because they are then automatically closed. Having said that, other people do like it so it's more of a personal choice I guess.

Reply:
agree, its personal choice, and I like automation so will keep it